src/job-exporter/src/nvidia.py

```diff
@@ -101,8 +101,13 @@ def parse_smi_xml_result(smi):
 
     result = {}
 
-    for gpu in gpus:
-        minor = gpu.getElementsByTagName("minor_number")[0].childNodes[0].data
+    for index, gpu in enumerate(gpus):
+        if os.getenv("LAUNCHER_TYPE") == "k8s":
+            # For pai k8s, the minor number doesn't match the NVIDIA_VISIBLE_DEVICES number,
+            # use nvidia-smi gpu sequence index instead
+            minor = index
+        else:
+            minor = gpu.getElementsByTagName("minor_number")[0].childNodes[0].data
```

Comment:
change `minor` to another name, like `gpu_index`?

Reply:
fixed